caduceus.yaml

```diff
@@ -351,6 +351,17 @@ sender:
 #   - 1000
 #   - 10000
 
+  # noPIDAction dictates what to do if PartnerIDs are missing or empty. "donotsend"
```

Comment:
Similar to tr1d1um, we want a boolean that tells us whether or not to enforce the partner ID check (with `false` being enforcing).